src/sentry/integrations/jira/integration.py

```diff
@@ -220,6 +232,17 @@ def update_organization_config(self, data):
                     unresolved_status=statuses["on_unresolve"],
                 )
 
+        if self.issues_ignored_fields_key in data:
+            ignored_fields_text = data.pop(self.issues_ignored_fields_key)
+            # While we describe the config as a "comma-separated list", users are likely to
+            # accidentally use newlines, so we explicitly handle that case. On page
+            # refresh, they will see how it got interpreted as `get_config_data` will
+            # re-serialize the config as a comma-separated list.
+            ignored_fields_list = filter(
+                None, [field.strip() for field in re.split(r"[,\n\r]+", ignored_fields_text)]
+            )
```

Comment:
```suggestion
            ignored_fields_list = [field.strip() for field in re.split(r"[,\n\r]+", ignored_fields_text) if field.strip() != ""])
```

Reply:
@BYK : I personally like `filter(None, ...)` like this instead of repeating the logic of `field.strip()` twice (and also prefer it over the (IMO) grosser double list comprehension: `[y for y in [x.strip() for x in ...] if y]`). I saw a couple other usages of it in the codebase, so assumed it was okay stylistically, but LMK if you feel strongly about this.

Reply:
Two things with using `filter` then:

1. ~~It returns an iterable instead of a list in python 3 and the variable name does not match it right now.~~ Looks like you have used our compat override per the lint rules, this is fine
2. You can then use a generator instead of an explicit list as its second argument:
```py
ignored_fields_list = filter(
    None, (field.strip() for field in re.split(r"[,\n\r]+", ignored_fields_text))
)
```